### DISCUSSION TITLE
SML: Publish a ready to use binary

### DISCUSSION THREAD

**Discussion by CFlix** [arendst/Tasmota#20220] [Q&A]
Hi,
is there something to be said against publishing a ready to use tasmota-smartmeter.bin.gz like for e.g. tasmota-sensors.bin.gz?
I think this feature gained a lot of attention especially with the balcony solar initiative going on in Germany as this is an easy way to monitor the power in a household. So to simplify the usage of this feature an officially precompiled binary would for sure attract even more users that currently hesitate to use it because they have to compile themselves or use pre-compiled binaries from unverified sources.

**Comment by Noschvie** [comment]
Feature Request: Precompiled Binaries with Smart Meter Interface / SML #18012

**Reply by sfromis** [reply]
This is already included in the "special" scripting binary available from
https://github.com/Jason2866/Tasmota-specials/tree/firmware/firmware/tasmota/other
Also available via the regular web installer as "unofficial" binaries. https://tasmota.github.io/install/

This was previously mentioned in https://github.com/arendst/Tasmota/discussions/17733 already linked from https://github.com/arendst/Tasmota/discussions/18012
